platform-api/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/model/impl/CommandImpl.java

```diff
@@ -73,9 +84,10 @@ public boolean equals(Object obj) {
             return false;
         }
         final CommandImpl command = (CommandImpl)obj;
-        return Objects.equals(name, command.name) &&
-               Objects.equals(commandLine, command.commandLine) &&
-               Objects.equals(type, command.type);
+        return Objects.equals(getName(), command.getName()) &&
```

Comment:
Previous version of the equals was pretty ok, you could just add a new line of `Objects.equals` and that's it, why did you add getters here?